packages/blob/src/index.ts

```diff
@@ -229,19 +235,25 @@ export async function list(
   };
 }
 
-function getApiUrl(): string {
-  return (
+function getApiUrl(pathname = ''): string {
```

Comment:
I understand it may take more time, wouldn't it better to have `https://blob.vercel-storage.com/v2`?

Reply:
I see that, for some reason I do not remember, we call the Vercel Blob API on put this way: blob.vercel-storage.com/pathname, so we can't have `v2` now here :D

Reply:
not that important as it's still a private api, if/when we are going to expose it, it could be a nicer interface